dll.py

```diff
@@ -0,0 +1,40 @@
+class Node(object):
+
+    def __init__(self, vertex, prev, next):
```

Comment:
suggestie :`def __init__(self, vertex: "Vertex", prev=None: "Vertex", next=None: "Vertex")`
1. maak `prev` en `next` optional
2. type checking voor vertices (tenzij we het ook voor andere dingen willen gebruiken)